Gems/Atom/Tools/MaterialCanvas/Assets/MaterialCanvas/GraphData/Nodes/MaterialOutputs/StandardPBR/MaterialGraphName_SurfaceEval.azsli

```diff
@@ -79,7 +79,7 @@ Surface EvaluateSurface_MaterialGraphName(
     surface.specularOcclusion = inSpecularOcclusion;
     surface.opacityAffectsSpecularFactor = o_opacity_affects_specular_factor;
 
-    CheckClipping(surface.alpha, 0.0);
+    CheckClipping(surface.alpha, 0.5);
```

Comment:
Should this be hardcoded like this? Does this value need to match the depth pass?

Reply:
For now, this PR is resolving a regression. The value will be configurable after I surface it on the material canvas node for this type. There is an existing task to rework opacity factor. In the hardcoded material types, the value for opacity factor came from a fixed material SRG, with a default value of 0.5. It was used inside GetAlphaAndClip. I will be noting and exposing more parameters as I come across them. I want to do it in such a way where I'm not littering all of the code with material canvas code generation blocks, which I will also be simplifying.